lectures/dynamic_programming/wald_friedman.md

```diff
@@ -423,7 +426,7 @@ function choice(p, d0, d1, L0, L1, c)
 end
 ```
 
-Next we solve a problem by finding the α, β values for the decision rule
+Next we solve a problem by finding the alpha, beta values for the decision rule
```

Comment:
by finding the `$\alpha, \beta$` values  since this is in text and should show latex